.eslintrc.js

```diff
@@ -1,30 +1,26 @@
 module.exports = {
-  globals: {
-    server: true,
-  },
   root: true,
   parser: 'babel-eslint',
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2018,
     sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
   },
   plugins: ['ember', 'prettier'],
-  extends: ['eslint:recommended', 'plugin:ember/recommended', 'plugin:prettier/recommended'],
+  extends: ['eslint:recommended', 'plugin:ember/octane', 'plugin:prettier/recommended'],
   env: {
     browser: true,
   },
   rules: {
-    'prettier/prettier': 'error',
-    'ember/order-in-components': 'error',
-    'ember/order-in-controllers': 'error',
-    'ember/order-in-routes': 'error',
-    'prefer-const': 'error',
-    'no-var': 'error',
+    'ember/use-ember-data-rfc-395-imports': 'error',
```

Comment:
Is there a reason why `prettier/prettier`, `prefer-const` and `no-var` were removed?

Reply:
I've simply used Ember CLI's 3.16 default eslintrc.js and just added the prettier and ember-data rules.